docs/app-check/default-providers.md

```diff
@@ -81,10 +81,17 @@ Future<void> main() async {
     webRecaptchaSiteKey: 'recaptcha-v3-site-key',
     // Default provider for Android is the Play Integrity provider. You can use the "AndroidProvider" enum to choose
     // your preferred provider. Choose from:
-    // 1. debug provider
-    // 2. safety net provider
-    // 3. play integrity provider
+    // 1. Debug provider
+    // 2. Safety Net provider
+    // 3. Play Integrity provider
     androidProvider: AndroidProvider.debug,
+    // Default provider for iOS/macOS is the Device Check provider. You can use the "AppleProvider" enum to choose
+        // your preferred provider. Choose from:
+        // 1. Debug provider
+        // 2. Device Check provider
+        // 3. App Attest provider
+        // 4. App Attest provider with fallback to Device Check provider (App Attest provider is only available on iOS 14.0+, macOS 14.0+)
```

Comment:
❤️  on the backwards compatible one. I did it for react-native-firebase because...I couldn't think of a better way to handle the iOS 14 minimum, just seems developer-friendly.

Reply:
It's a good idea I think 😄